source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
```

Comment:
How about a general setter which takes a FloatMatrix/FloatPoint2D/etc. and (x,y) for the top-left location (for example) in the matrix and replaces the values starting from there?